src/scalper_utils.ts

```diff
@@ -92,6 +92,36 @@ export function orderSplice(
   return spliceFactor;
 }
 
+// Splice delta hedge orders if available mango liquidity not supportive
+export function orderSpliceOpenBook(
```

Comment:
what does this function return? name should be descriptive

Reply:
Same as the mango version. Just a divisor on the total hedge delta if necessary. Splice similar enough name as divisor to leave